scripts/percona-release.sh

```diff
@@ -355,6 +391,7 @@ if [[ ${COMMANDS} != *${1}* ]]; then
   exit 2
 fi
 #
+check_repo_availability $@
```

Comment:
This function still doesn't solve the issue from PREL-42
```

[23:19:10 vagrant@localhost:~] $ yum list all | grep -i percona
http://repo.percona.com/pxc-80/yum/release/6/RPMS/noarch/repodata/repomd.xml: [Errno 14] PYCURL ERROR 22 - "The requested URL returned error: 404 Not Found"
Trying other mirror.
Error: Cannot retrieve repository metadata (repomd.xml) for repository: pxc-80-release-noarch. Please verify its path and try again

```

Reply:
fixed